Xamarin.Forms.Core/ActivityIndicator.cs

```diff
@@ -1,5 +1,9 @@
 using System;
+using Xamarin.Forms;
 using Xamarin.Forms.Platform;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-activityindicator-color", typeof(ActivityIndicator), nameof(ActivityIndicator.ColorProperty))]
```

Comment:
we need to reach an agreement about moving all the StyleProperties to the class (or interface) they apply to, or to keep them all in Properties.cs

Reply:
You're the man. I swung this convention past @jassmith and he didn't have objection. However, I'm sure neither of us are wed to this convention.  We'll just agree to do whatever you want. 

I put them into the renderer file because the vender specific css property names are bound quite tightly to the renderer. This as opposed to the standard css names which work across many different renderers by binding to BPs in base classes.